### PR TITLE
NO-TICKET - Update production CI

### PR DESCRIPTION
**Pull request by davran1001**
## Jira Ticket

No Jira Ticket
## Description of Work

[ what work was done in this ticket ]

## Screenshots (Optional)

Screenshot #1 | Screenshot #2 | Screenshot #3
:------------:|:-------------:|:------------:
[Paste Pic 1] | [Paste Pic 2] | [Paste Pic 3]

### CODE REVIEW

**Review by kabramovp2p — APPROVED** [review]

File: .github/workflows/android-release-workflow.yml [hunk 1]
```diff
@@ -76,43 +83,55 @@ jobs:
...
           amplitude-secret: ${{ secrets.AMPLITUDE_KEY }}
 
-      - name: Create and fill cert.pem certificate for SSL pinning
+      - name: "Create cert.pem file"
         env:
           SSL_CERTIFICATE: ${{ secrets.SSL_CERTIFICATE }}
         run: echo $SSL_CERTIFICATE -> app/src/main/res/raw/cert.pem
 
-      - name: Create and fill sentry.properties
+      - name: "Create sentry.property file"
```
Comment (gslevinkov) [inline .github/workflows/android-release-workflow.yml:91]:
```suggestion
      - name: "Create sentry.properties file"
```

File: .github/workflows/android-release-workflow.yml [hunk 2]
```diff
@@ -76,43 +83,55 @@ jobs:
...
+      - name: "Setup signing.properties"
+        run: echo "${{ secrets.KEYSTORE_SECRET }}" | base64 --decode -> p2p-wallet.jks
 
-      - uses: ./.github/actions/setup-gradle
 
-      - name: Upload artifact to Firebase
+      - name: "Setup gradle"
+        uses: ./.github/actions/setup-gradle
+
+      - name: "Build Bundle artifact"
```
Comment (gslevinkov) [inline .github/workflows/android-release-workflow.yml:104]: it's not only bundle artifact, but it's APK too

Comment (gslevinkov) [inline .github/workflows/android-release-workflow.yml:104]:
```suggestion
      - name: "Build APK and AAB artefact"
```

File: .github/workflows/android-release-workflow.yml [hunk 3]
```diff
@@ -76,43 +83,55 @@ jobs:
...
-      - name: Upload APK to Slack
-        uses: ./.github/actions/setup-slack
-        with:
-          bot-token: ${{ secrets.SLACK_BOT_TOKEN }}
-          channel-id: ${{ secrets.SLACK_CHANNEL_ID }}
-          build-type: "release"
-          artifact-type: "apk"
-
-      - name: Upload Bundle to Slack
+      - name: "Upload APK to Slack Channel"
```
Comment (gslevinkov) [inline .github/workflows/android-release-workflow.yml:117]:
```suggestion
      - name: "Upload AAB to Slack Channel"
```

File: .github/workflows/android-release-workflow.yml [hunk 4]
```diff
@@ -76,43 +83,55 @@ jobs:
...
         uses: ./.github/actions/setup-slack
         with:
           bot-token: ${{ secrets.SLACK_BOT_TOKEN }}
           channel-id: ${{ secrets.SLACK_CHANNEL_ID }}
           build-type: "release"
           artifact-type: "bundle"
+
+      - name: "Upload changelog, and Jira ticket to slack channel"
```
Comment (gslevinkov) [inline .github/workflows/android-release-workflow.yml:125]:
```suggestion
      - name: "Upload changelog and Jira ticket to Slack channel"
```

File: .github/workflows/android-release-workflow.yml [hunk 5]
```diff
@@ -76,43 +83,55 @@ jobs:
...
+        run: |
+          cd bash_scripts
+          value=`cat timestamp.txt`
+          bash slack_deploy_jira_ticket.sh ${{ secrets.SLACK_BOT_TOKEN }} ${{ secrets.SLACK_CHANNEL_ID }} $value
+          bash slack_deploy_changelog.sh ${{ secrets.SLACK_BOT_TOKEN }} ${{ secrets.SLACK_CHANNEL_ID }}
+          cd ..
+
+      - name: "Send finish deploy message to slack"
```
Comment (gslevinkov) [inline .github/workflows/android-release-workflow.yml:133]:
```suggestion
      - name: "Send finish deploy message to Slack"
```

File: .github/workflows/android-release-workflow.yml [hunk 6]
```diff
@@ -1,22 +1,21 @@
 on:
   pull_request:
     branches:
-      - release
+      - main
```
Comment (gslevinkov) [inline .github/workflows/android-release-workflow.yml:4]: what should we do with hotfix branches that would fix bugs on release build?
the branch would be feature/PWN-XXX to -> release/vX.X.X, how can we get the release build there?

**Review by gslevinkov — APPROVED** [review]

**Pull request merged**